pages/data/explore.vue

```diff
@@ -45,114 +62,172 @@
 </template>
 
 <script>
-import WoudcBlurb from '~/components/WoudcBlurb'
-import WoudcSelect from '~/components/WoudcSelect'
-
-const now = new Date()
-
 export default {
-  components: {
-    'woudc-blurb': WoudcBlurb,
-    'woudc-select': WoudcSelect
-  },
   data() {
+    const now = (new Date()).getFullYear()
```

Comment:
This should be placed within data() { return {...} }
```suggestion
    now: (new Date()).getFullYear(),
```

Also, you may want to consider using vue-moment: https://www.npmjs.com/package/vue-moment

It has far more utility in date/time manipulation, parsing and display than the standard `Date()` class. With vue-moment, you can do something like: 
```js
let now = this.$moment()
now.add(1, 'y')
const maxYearPlus1 = now.format('yyyy') // 2021
```